CHANGELOG.md

```diff
@@ -1213,6 +1213,7 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 - Remove ODA fields from the non-ODA CSV upload template.
 - Move the comments column to the end of all the CSV templates.
 - Update Ruby to 3.1.3
+- ODA only attributes are blank on non-ODA activities.
```

Comment:
Sorry, most annoying comment I've ever written, but let's remove `.`s at the end of the Changelog lines here

Reply:
Ha, that was on my mind too @Gweaton

Reply:
Is this a rule I don't know about, or just to keep them matching? I thought a changelog was a list of sentences, not titles (if that makes sense?).

Also while we are on the subject, I think the changelog here is upside down https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first not sure if it's worth flipping it.

Reply:
More to keep them matching the rest of the file. Yeah the upside-downness is a bit annoying, but we haven't really been too bothered by it to prioritise changing it just yet.

Reply:
Agree that it's more consistency than anything else, but I guess given each point is written as a present tense declarative, they're quite similar to the first line of a commit. They are legit sentences though

I used to be a big adherent of consistently using full stops everywhere, but eventually I [dropped it at the end of paragraphs](https://forum.wordreference.com/threads/omitting-a-full-stop-at-end-of-sentence-in-paragraphs.2362088) in all but the most formal/publication-y writing because of the connotations it has in internet/messaging contexts of a serious tone

Reply:
Interesting! 

I've got into the habit of using an exclamation mark every other sentence for the same reason!

Reply:
Same!